### DISCUSSION TITLE
Geometry optimization vs Molecular Dynamics

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#756] [Q&A] [answered]
Hello,

I started from the same structure for both type of calculations: geometry optimization (GEO) and molecular dynamics (MD). In both, I used the same hamiltonians/methods with the same accuracy.

At the end of both simulations, the final structures are very different. It looks like in MD the structure is more flexible than when running GEO, suffering more structural changes.

**If I would like to study the interaction between two systems, which method should I use?**

I appreciate any help about this.

Thanks in advance.

Camps

**Comment by MtoLStoN** [comment, marked as answer]
Hi,

You are mixing up two completely different approaches here. A Geometry optimization utilizes some algorithm (e.g., the steepest descent) to reach a local minimum structure on the respective [potential energy surface](https://chem.libretexts.org/Bookshelves/Physical_and_Theoretical_Chemistry_Textbook_Maps/Physical_Chemistry_%28LibreTexts%29/30%3A_Gas-Phase_Reaction_Dynamics/30.10%3A_The_Potential-Energy_Surface_Can_Be_Calculated_Using_Quantum_Mechanics).

On the other hand, a molecular dynamics simulation simulates, as the name suggests, the molecular dynamics of the system. You can use this to analyze atoms' and molecules' movements at a specific temperature. Thinking about the PES picture, you can use this method to explore the PES. A molecular dynamics simulation is done for a specific time frame (e.g., 500ps) with a particular time step (e.g., 2 fs) and an optional dump step (e.g., 50 fs), at which the structure reached is written in a trajectory file. However, these structures are usually not local minimum structures of the system.

These methods are often combined for larger, more flexible systems to reach several local minima on the PES (conformers) or even find the global minimum (by energy screening). 

So, if you want to compare the interactions between two systems, you should always compare the lowest energy conformers of the two systems. If you have relatively rigid systems, geometry optimization will suffice. However, if your system is more flexible, I suggest combining [CREST](https://github.com/crest-lab/crest) and [CENSO](https://github.com/grimme-lab/CENSO) to identify the right conformers.
